gwt-jackson/src/test/java/com/github/nmorel/gwtjackson/jackson/mapper/DifferentTypeGetterJacksonTest.java

```diff
@@ -0,0 +1,20 @@
+package com.github.nmorel.gwtjackson.jackson.mapper;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.github.nmorel.gwtjackson.jackson.AbstractJacksonTest;
+import com.github.nmorel.gwtjackson.shared.mapper.DifferentTypeGetterTester.DifferentTypeGetterBean;
+
+public class DifferentTypeGetterJacksonTest extends AbstractJacksonTest {
+
+	@Test
+	public void testDifferentTypeGetter()  {

```

Comment:
Why don't you use `DifferentTypeGetterTester` ?

Reply:
I only wanted to test the generation of serializer/deserializer, but I could as well have used the same tester. It is more consistent with the other tests also. Changing this :)